cypress.config.js

```diff
@@ -1,19 +1,27 @@
 const { defineConfig } = require('cypress');
 const { devServer } = require('@cypress/webpack-dev-server');
 const webpackConfig = require('./config/cypress.webpack.config.js');
+const codeCoverageTask = require('@cypress/code-coverage/task');
 
 module.exports = defineConfig({
   viewportWidth: 1000,
   viewportHeight: 660,
   video: false,
   component: {
+    env: {
+      BABEL_ENV: 'component',
+    },
```

Comment:
Do we need it here since you already provide this env variable at package.json script?

Reply:
Removed it